caravel/assets/javascripts/SqlLab/components/QueryHistory.jsx

```diff
@@ -9,12 +9,17 @@ import { Alert } from 'react-bootstrap';
 
 const QueryHistory = (props) => {
   const activeQeId = props.tabHistory[props.tabHistory.length - 1];
-  const queries = props.queries.filter((q) => (q.sqlEditorId === activeQeId));
-  if (queries.length > 0) {
+  var queries_array = []
```

Comment:
this is probably failing in the linter, should be `const queriesArray = [];`
this is the style guide we are using.